packages/postcss-normalize-string/src/index.js

```diff
@@ -259,7 +259,14 @@ export default postcss.plugin('postcss-normalize-string', (opts) => {
           return;
         }
 
-        const newValue = normalize(node[param], preferredQuote);
+        let newValue;
+
+        // Ref https://github.com/cssnano/cssnano/issues/927
+        if (Array.isArray(node[param])) {
+          newValue = normalize(node[param][0], preferredQuote);
```

Comment:
What about `[1]`/`[2]`/etc?

Reply:
I have tested a few cases, the array length was always one only. 

I don't know why the value is being converted into an array of the same content.

Reply:
You need to search why it happens